db/migrate/20160523202611_create_profiles.rb

```diff
@@ -0,0 +1,13 @@
+class CreateProfiles < ActiveRecord::Migration
+  def change
+    create_table :profiles do |t|
+
+      t.integer :user_id, index: true
+      t.string  :user_full_name
```

Comment:
We don't need this field. User will have `first_name` and `last_name` in `users` table.